pkg/errors/errors.go

```diff
@@ -87,6 +87,9 @@ const (
 	// ErrorCode_OTHERS 99xx
 	ErrorCode_RESOURCE_EXHAUSTED  ErrorCode = 9901
 	ErrorCode_RESOURCE_CAN_NOT_OP ErrorCode = 9902
+
+	ErrorCode_Unauthenticated  ErrorCode = 9910
```

Comment:
the error codes for ErrorCode_Unauthenticated and ErrorCode_PermissionDenied should not be the same

Reply:
modify